modules/cactus-web/src/MenuBar/MenuBar.tsx

```diff
@@ -273,34 +352,32 @@ export { TypedMenuBar as MenuBar, MenuBarList, MenuBarItem }
 
 export default TypedMenuBar
 
-const Nav = styled.nav`
+const Nav = styled.nav<MenuBarProps>`
   width: 100%;
   display: flex;
   flex-flow: row nowrap;
   align-items: stretch;
   position: relative;
   outline: none;
-  ${(p) => textStyle(p.theme, 'small')};
-  ${(p) => p.theme.colorStyles.standard};
-  box-shadow: inset 0 -${(p) => border(p.theme, 'lightContrast').replace('solid', '0')};
-
+  [role='menubar'] {
+    padding-bottom: 35px;
+    margin-bottom: -35px;
+  }
   [role='menubar'] > li > [role='menuitem'] {
     white-space: nowrap;
-    padding: 20px 8px;
-    border: ${(p) => border(p.theme, 'transparent')};
-    border-bottom-color: ${(p) => p.theme.colors.lightContrast};
+    padding: 24px 16px;
+    ${(p) => textStyle(p.theme, 'body')};
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+  [role='menubar'] > li {
     &:hover,
     &[aria-expanded='true'] {
-      color: ${(p) => p.theme.colors.callToAction};
-      border-bottom-color: ${(p) => p.theme.colors.callToAction};
-    }
-    &:focus {
-      border: ${(p) => border(p.theme, 'callToAction')};
-    }
-    &[aria-current='true'] {
-      font-weight: bold;
+      z-index: 100;
+      box-shadow: ${shadowTypes[2]} ${(p) => p.theme.colors.transparentCTA};
```

Comment:
What you're doing here is equivalent to the `boxShadow` helper that already exists.
```suggestion
      ${(p) => boxShadow(p.theme, 2)};
```